README.md

```diff
@@ -12,6 +12,17 @@ also be discussed in the forum at https://forum.snapcraft.io.
 Our localization platform is Launchpad, translations are managed at
 https://translations.launchpad.net/ubuntu/+source/subiquity/
 
+To update translation template in launchpad:
+ * execute $ make i18n
```

Comment:
I thought this was handled by the default make target, why do we need an explicit make invocation here?

also are there any bits that need committed to the git repo after running this command, or are they all build artifacts not in git?

Reply:
Default target is all, which depends on `dryrun`, `probert`, `i18n`. Thus the default target is a larger superset which requires more things to be installed on the system. The actual minimum is to run the `i18n` target, or any of the others. What it does is simply invoke `python3 setup.py build` which has all the i18n hooks already hooked up. I'm specifying the minimal requirements here.

`.pot` changes timestamp on every build, thus i found it annoying to keep committed in git. Thus both `*.pot` and `*.mo` are ignored in `.gitignore`. The only thing that remains committed in git are the `po` translation files exported from the launchpad and documented on how to update those further down in the text.

Reply:
should we not perhaps make this a dependency of the source package's clean target, then, so it gets done automatically on a source upload?

Reply:
oooh, yes.